resources/frontend_shared/static_viz_interface.js

```diff
@@ -25,8 +25,8 @@ function row_chart(settings, data, colors) {
 function combo_chart(series, settings, colors) {
   // Thinking of combo as similar to multiple, although they're different in BE
   return StaticViz.RenderChart("combo-chart", {
-    series: JSON.parse(series),
-    settings: JSON.parse(settings),
+    multipleSeries: JSON.parse(series),
```

Comment:
noted that on the BE I use the parameters called `series-seqs` and `settings-seqs` just to follow what's already defined in `render-multiple-lab-chart` e.g. `row-seqs` represents rows from each question in a dashcard, so if a dashcard contains 3 questions `row-seqs` will have 3 elements with each element/question contains all series in that question.